lib/qmp.ml

```diff
@@ -37,12 +37,18 @@ type vnc = {
   host    : string;
 }
 
+type xen_platform_pv_driver_info = {
```

Comment:
I think "info" can be dropped here (and below) without any loss of clarity. Is the name coming from somewhere outside such that it would be unwise not to use it?